website/app_test.js

```diff
@@ -118,6 +119,7 @@ test(function createExecTimeColumnsIrregularData() {
     ["cold_relative_import", 0, 0]
   ]);
 });
```

Comment:
Reviewing on my phone so difficult to see commit messages - but I hope there’s explanation there for what’s wrong here..

Reply:
Ah, I guess it is #882 yesterday that replaces 0 with null... Tests were not updated...